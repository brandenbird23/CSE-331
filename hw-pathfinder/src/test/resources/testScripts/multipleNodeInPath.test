# Test for when multiple nodes connect the path

CreateGraph graph

AddNode graph A
AddNode graph B
AddNode graph C
AddNode graph D

AddEdge graph A B 5.0
AddEdge graph B C 7.0
AddEdge graph C D 3.0

FindPath graph A D