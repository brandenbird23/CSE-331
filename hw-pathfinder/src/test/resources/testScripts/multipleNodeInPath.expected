# Test for when multiple nodes connect the path

created graph graph

added node A to graph
added node B to graph
added node C to graph
added node D to graph

added edge 5.000 from A to B in graph
added edge 7.000 from B to C in graph
added edge 3.000 from C to D in graph

path from A to D:
A to B with weight 5.000
B to C with weight 7.000
C to D with weight 3.000
total cost: 15.000